crates/optimism/trie/src/metrics.rs

```diff
@@ -248,69 +235,87 @@ impl<C: OpProofsTrieCursor> OpProofsTrieCursor for TrieCursorWithMetrics<C> {
         })
     }
 
+    #[inline]
```

Comment:
added inline attributes so that benchmarks wouldn't be effected by these function calls from wrapping types at least when not built with maxperf